website/i18n/en.json

```diff
@@ -17,18 +17,33 @@
       "api/basic-crawler": {
         "title": "BasicCrawler"
       },
+      "api/browser-crawler": {
+        "title": "BrowserCrawler"
+      },
       "api/browser-plugin": {
         "title": "BrowserPlugin"
       },
+      "api/cache-container": {
+        "title": "CacheContainer"
+      },
       "api/cheerio-crawler": {
         "title": "CheerioCrawler"
       },
       "api/configuration": {
         "title": "Configuration"
       },
+      "api/cookie-parse-error": {
+        "title": "CookieParseError"
+      },
       "api/dataset": {
         "title": "Dataset"
       },
+      "api/exports": {
+        "title": "exports"
```

Comment:
Not sure what this is, but it feels wrong :)

Reply:
i dont think it matters much, this is just some translation file

(but its indeed weird)

Reply:
I don't even know where it came from. I remember running only `generate` and `start` commands.